packages/nova-base-components/lib/users/UsersProfile.jsx

```diff
@@ -17,7 +17,7 @@ const UsersProfile = ({user}, {currentUser}) => {
     <div className="page users-profile">
       <Telescope.components.HeadTags url={Users.getProfileUrl(user, true)} title={Users.getDisplayName(user)} description={user.telescope.bio} />
       <h2 className="page-title">{Users.getDisplayName(user)}</h2>
-      <p>{user.telescope.bio}</p>
+      {user.telescope.htmlBio ? <p>{Telescope.utils.stripHTML(user.telescope.htmlBio)}</p> : null }
```

Comment:
That's true that we need `htmlBio`, because it's the actual [published field](https://github.com/TelescopeJS/Telescope/blob/master/packages/nova-users/lib/schema.js#L117), and not bio. 

However, as we authorize the `bio` to be written in Markdown and it's already sanitized in the [collection hooks](https://github.com/TelescopeJS/Telescope/blob/master/packages/nova-users/lib/callbacks.js#L33), we should keep the html created.

Keeping the condition to show or not `user.telescope.htmlBio`, could you display it like `htmlBody` in `PostsPage` please? https://github.com/TelescopeJS/Telescope/blob/master/packages/nova-base-components/lib/posts/PostsPage.jsx#L17